docs/conf.py

```diff
@@ -0,0 +1,36 @@
+"""
+A configuration file for RTD with Sphinx
+"""
+
+
+# -- Path setup --------------------------------------------------------------
+
+import sys, os
+
+sys.path.append(".")
+
+
+# -- Project information -----------------------------------------------------
+
+project = "Kart"
+copyright = "2022, Kart Contributors"
+author = "Kart Contributors"
+release = "0.11.1dev0"
```

Comment:
Need to add this into `HOWTO-RELEASE.md` so the version-number bump doesn't get missed in releases.